### ISSUE TITLE
Model Output

### ISSUE THREAD

**Issue by 464hee** [engrodawood/HiGGsXplore#2]
#########
SlideGraph -platt.py: Normalise classifier output scores to a probability value########

I saw in SlideGraph the probability of converting the output of the model into categories, and I was wondering if HiGGsXplore has a corresponding code that makes it possible to calculate metrics such as F1

**Comment by engrodawood** [comment]
The prediction score for each class by the model can be used for computing F1-score. As per this example: https://scikit-learn.org/stable/modules/generated/sklearn.metrics.f1_score.html if you simple pass the model prediction score per-class and the corresponding binary label it should be fine. 

If you need probability you can use platt.py for each output which will do the job.
